project-codeflare/codeflare-operator
# Added all-in-one command to Makefile

# Issue link


closes https://issues.redhat.com/browse/RHOAIENG-6661
# What changes have been made

This make command will try to delete all installed RHOAI related operators and install them again.
This includes NVIDIA GPU Operator, NFD Operator, RHOAI Operator, Service Mesh Operator

By default the latest version of RHOAI is installed with this command. To install ODH instead an additional env var is required.
# Verification steps



* Create an OpenShift Cluster of any type
* Run `make all-in-one`
* Ensure all the required Operators are installed correctly
* To install ODH instead just run `make all-in-one -e USE_RHOAI=false`
* To delete all related Operators run `make delete-all-in-one -e USE_RHOAI=true/false`
## Checks
- Testing Strategy
